Modules/Remote/MorphologicalContourInterpolation.remote.cmake

```diff
@@ -14,5 +14,5 @@ IEEE Trans Biomed Eng. 2008 Aug;55(8):2022-38. doi: 10.1109/TBME.2008.921158.
 This work is supported by NIH grant R01 EB014346
 'Continued development and maintenance of the ITK-SNAP 3D image segmentation software'."
   GIT_REPOSITORY ${git_protocol}://github.com/KitwareMedical/ITKMorphologicalContourInterpolation.git
-  GIT_TAG 608853faf597c38132eaff8586f1f9c5c31f70aa
+  GIT_TAG ITKv5
```

Comment:
And until these tags are added, enabling remote modules will cause errors? That's certain to draw attention :smile:

Reply:
These are not intended for long term use.  I think they are being pushed now.  I'll test on a secondary machine soon.

Reply:
I meant, if anyone tries building your PR he will be clearly reminded that remote modules should not be turned on :smile: